test/context.test.js

```diff
@@ -25,6 +25,15 @@ describe('Context', function () {
   })
 
   describe('repo', function () {
+    it('returns error for context.repo()', function () {
```

Comment:
A tiny change, but would you mind moving this test to the bottom of this `describe` block? I find it easier to understand the functionality when tests are presented by an order of "this is how it works, this is how errors are handled."

Reply:
You want me to change the name of test? I don't quite understand this

Reply:
Sure - I meant to move this test after the other `repo` tests:

```js
describe('repo', function () {
  it('returns error for context.repo()', function () {
    context.payload.repository = undefined
    try {
      context.repo()
    } catch (e) {
      expect(String(e)).toEqual('Error: Sorry not available for this event')
    }
  })

  it('..')
  it('..')
})
```
to
```js
describe('repo', function () {
  it('..')
  it('..')

  it('returns error for context.repo()', function () {
    context.payload.repository = undefined
    try {
      context.repo()
    } catch (e) {
      expect(String(e)).toEqual('Error: Sorry not available for this event')
    }
  })
})
```

Reply:
Done. :+1: